frontend/src/store/reducers/index.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable prettier/prettier */
+import {
+	REGISTER_USER,
+	LOGIN_USER,
+	LOGOUT_USER,
+	ADD_ARTICLE,
+	DELETE_ARTICLE,
+	FETCH_DATA,
+	DATA_FETCHED,
+	ERROR
+} from '../actions';
+
+const initialState = {
```

Comment:
Missing initialState for articles, though you should move all the article and user reducer stuff to their own files, then use combineReducers to bring it all together in reducers/index.js.